frontend/src/components/loading/spinner.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { onMount } from "svelte"
+
+  const LONG_LOAD_TIME = 3000
+  let tooLongLoad = false
+
+  onMount(async () => {
+    setTimeout(() => (tooLongLoad = true), LONG_LOAD_TIME)
+  })
+</script>
+
+<div class="flex justify-center items-center">
+  <div class="animate-spin rounded-full h-32 w-32 border-b-8 border-neutral" />
+</div>
+{#if tooLongLoad}
+  <h2 class="flex justify-center pt-6">The request is taking unusally long</h2>
+  <br />
+  <div class="flex justify-center">
+    <button
+      on:click={() => window.history.back()}
+      id="goback"
+      type="button"
+      class="btn"
+    >
+      Back to safety
+    </button>
+  </div>
```

Comment:
Could this be changed to our error component?

Reply:
Technically yes it is a good idea, but dont  think it should happen this PR since the Error component would have to be redone.

Reply:
Close if you agree

Reply:
Yeah its fine for now, we can have a discussion about at some other time.